modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceOrders.macro

```diff
@@ -22,32 +22,6 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
-	macro addTermsAndCondition {
```

Comment:
why did you change the macros?

Reply:
I don't know, I don't think I did it intentionally

Reply:
i can't find the commit where i would edit the file